bin/node-sass

```diff
@@ -182,6 +182,7 @@ function getEmitter() {
  */
 
 function getOptions(args, options) {
+  var cssDir, sassDir, file, mapDir;
```

Comment:
Yeah, if we get to just versions that support ES2015, then we can use stuff like `let` and `const`, but I don't think it's work adding in something like Babel just to get that right now